src/components/network/network.js

```diff
@@ -66,7 +100,16 @@ genotet.NetworkView = function(viewName, params) {
         this.renderer.update();
         break;
       case 'gene':
-        this.loader.updateGenes(data.method, data.regex);
+        this.loader.updateGenes(data.method, data.inputGenes, data.isRegex);
+        this.renderer.dataLoaded();
```

Comment:
dataLoaded involves prepareData_(). Are you sure this is intended? Same for the cases below.